xapian-applications/omega/worker_comms.h

```diff
@@ -22,6 +22,11 @@
 #include <cstdio>
 #include <string>
 
+enum assistant_msg {
+    MSG_OK,
+    MSG_NON_FATAL_ERROR,
+    MSG_FATAL_ERROR
+};
 /// Read a string from the file descriptor @a f and storage it in @a s
```

Comment:
I've added a blank like after the enum definition above (I only noticed this during merging)